cfpb/consumerfinance.gov
# Rename `o-table-wrapper--scrolling` to `o-table--scrolling`

Pull in https://github.com/cfpb/design-system/pull/2068

## Changes

- Rename `o-table-wrapper--scrolling` to `o-table--scrolling`


## How to test this PR

1. Visit ROUT and perform an address search and ensure the results scrolling table still scrolls.
